install/centos-installation-guide.md

```diff
@@ -0,0 +1,49 @@
+# Installing Kata Containers on CentOS
```

Comment:
Hi @egernst - please can you get https://github.com/kata-containers/documentation/pull/106 landed so that @GabyCT  can add a new bullet for CentOS to the installation README.

Reply:
Hi @GabyCT - this PR has now landed so please rebase and add CentOS to `install/README.md` on this PR.

Reply:
@jodh-intel done :) thanks for everything